src/components/CurrentTask/CurrentTask.js

```diff
@@ -38,14 +44,23 @@ export default function CurrentTask() {
 							<p>
 								{Math.floor(time / 60)}:{String(time % 60).padStart(2, '0')} min
 							</p>
+							<Clock role="img" alt="analoge Darstellung des Timers">
+								<Time
+									style={{
+										background: `conic-gradient(red ${time * 0.1}deg, blue 0 ${
+											tasks[0].time * 6
+										}deg, transparent ${tasks[0].time * 6}deg)`,
+									}}
+								></Time>
+							</Clock>
 							<Button onClick={() => setTimerIsRunnig(!timerIsRunnig)}>
 								{timerIsRunnig ? 'Pause' : 'Start'}
 							</Button>
 						</>
 					) : (
 						<p className="over">Fertig!</p>
```

Comment:
I would use a `span` element here as this is not really a text paragraph. And as already discussed it would be nicer to use here a styled component as well instead of `className`.